docs/data/pages.ts

```diff
@@ -0,0 +1,31 @@
+import type { MuiPage } from '@mui/monorepo/docs/src/MuiPage';
+
+const pages: MuiPage[] = [
+  {
+    pathname: '/toolpad',
+    title: 'Overview',
+    icon: 'DescriptionIcon',
+  },
+  {
+    pathname: '/toolpad/getting-started',
+    title: 'Getting Started',
```

Comment:
Sentence case convention https://www.notion.so/mui-org/Style-guide-2a957a4168a54d47b14bae026d06a24b#651fc5bd9b3f4faf9f23f04bb9cb7a6b

```suggestion
    title: 'Getting started',
```

Otherwise, it might work without anything too

```suggestion
```